OracleUnifiedDirectorySM/dockerfiles/12.2.1.4.0/Dockerfile

```diff
@@ -0,0 +1,149 @@
+#
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at 
+# https://oss.oracle.com/licenses/upl.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle Unified Directory Service Manager (OUDSM)
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# See oud.download file 
+# Also see patches.download file in the patches directory
+
+# ------------------------------------------------------------
+# Pull base image
+# ---------------
+FROM oracle/fmw-infrastructure:12.2.1.4.0 as base
+
+# Maintainer
+# ----------
+LABEL maintainer="OUD Team"
+
+#
+# Environment variables required for this build (do NOT change)
+# -------------------------------------------------------------
+ENV FMW_OUD_JAR=fmw_12.2.1.4.0_oud.jar \
+    BASE_DIR=/u01 \
+    ORACLE_HOME=/u01/oracle \
+    SCRIPT_DIR=/u01/oracle/container-scripts \
+    PROPS_DIR=/u01/oracle/properties \
+    PATCHES_BASE_DIR=/tmp/oracle_patches \
+    VOLUME_DIR=/u01/oracle/user_projects \
+    DOMAIN_NAME="${DOMAIN_NAME:-base_domain}" \
+    ADMIN_USER="${ADMIN_USER:-}" \
+    ADMIN_PASS="${ADMIN_PASS:-}" \
+    DOMAIN_ROOT="${DOMAIN_ROOT:-/u01/oracle/user_projects/domains}" \
+    DOMAIN_HOME="${DOMAIN_ROOT:-/u01/oracle/user_projects/domains}"/"${DOMAIN_NAME:-base_domain}" \
+    ADMIN_PORT="${ADMIN_PORT:-7001}" \
+    ADMIN_SSL_PORT="${ADMIN_SSL_PORT:-7002}" \
+    DOMAIN_TYPE="oudsm" \
+    USER_MEM_ARGS=${USER_MEM_ARGS:-"-Djava.security.egd=file:/dev/./urandom"} \
+    JAVA_OPTIONS="${JAVA_OPTIONS} -Dcom.sun.jndi.ldap.object.disableEndpointIdentification=true" \
+    PATH=$PATH:/usr/java/default/bin:/u01/oracle/oracle_common/common/bin:/u01/oracle/wlserver/common/bin:/u01/oracle/container-scripts
+
+#
+# Creation of User, Directories and Installation of OS packages
+# ----------------------------------------------------------------
+USER root
+RUN mkdir -p ${BASE_DIR} && \
+    mkdir -p ${PATCHES_BASE_DIR}/opatch_patch && \
+    mkdir -p ${PATCHES_BASE_DIR}/patches && \
+#    useradd -b ${BASE_DIR} -d ${ORACLE_HOME} -m -s /bin/bash oracle && \
```

Comment:
Don't leave commented out commands in the `Dockerfile`. If you don't need it, remove it.

Reply:
Removed